{"name": "test_private_repository_access", "status": "failed", "statusDetails": {"message": "AssertionError: Failed to access private repository with valid token\nassert 401 == 200\n +  where 401 = <Response [401]>.status_code", "trace": "def test_private_repository_access():\n        \"\"\"Step 3: Verify access to the private repository using valid authentication.\"\"\"\n        print(\"\\nðŸ”¹ Running: test_private_repository_access\")\n        response = api.get_repository(OWNER, PRIVATE_REPO_NAME)\n>       assert response.status_code == 200, \"Failed to access private repository with valid token\"\nE       AssertionError: Failed to access private repository with valid token\nE       assert 401 == 200\nE        +  where 401 = <Response [401]>.status_code\n\ntests\\api_tests\\test_authentication_api.py:46: AssertionError"}, "description": "Step 3: Verify access to the private repository using valid authentication.", "attachments": [{"name": "stdout", "source": "db8835d5-2968-43f3-b4ad-be769938bb2e-attachment.txt", "type": "text/plain"}], "start": 1742175257545, "stop": 1742175257821, "uuid": "48f6007f-a444-455a-b646-0529d74c0c9b", "historyId": "8b3dde7a1c33b8df361c2f8b8d0b49c2", "testCaseId": "8b3dde7a1c33b8df361c2f8b8d0b49c2", "fullName": "tests.api_tests.test_authentication_api#test_private_repository_access", "labels": [{"name": "parentSuite", "value": "tests.api_tests"}, {"name": "suite", "value": "test_authentication_api"}, {"name": "host", "value": "BHUVANSPC"}, {"name": "thread", "value": "9344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.test_authentication_api"}]}